nextcord/state.py

```diff
@@ -2294,6 +2295,23 @@ def parse_auto_moderation_action_execution(self, data) -> None:
             "auto_moderation_action_execution", AutoModerationActionExecution(data=data, state=self)
         )
 
+    def parse_guild_audit_log_entry_create(self, data) -> None:
+        guild = self._get_guild(int(data["guild_id"]))
+        user_id = int(data["user_id"])
+        user = self.get_user(user_id)
+
+        if guild is not None and user is not None:
+            entry = AuditLogEntry(
+                auto_moderation_rules={}, users={user_id: user}, data=data, guild=guild
+            )
+            self.dispatch("guild_audit_log_entry_create", entry)
+        else:
+            _log.debug(
+                "guild_audit_log_entry_create wasn't dispatched because the guild (%s) and/or user (%s) are None!",
+                repr(guild),
```

Comment:
`%r` I believe can be used, means the `repr` does not need to be executed if `logging.DEBUG` is not being used.